BUILD

```diff
@@ -1,37 +1,73 @@
 load("@rules_python//python:defs.bzl", "py_runtime_pair")
 load("@//tools:stamp.bzl", "stamp_build_setting")
-load("@pip_dependencies//:requirements.bzl", "entry_point")
+load("@aspect_bazel_lib//lib:expand_make_vars.bzl", "expand_template")
 
-# Needed by the custom toolchain
-# Can remove if we siwtch back to the canonical one
-"""
-py_runtime(
-    name = "python3_runtime",
-    files = ["@python_interpreter//:files"],
-    interpreter = "@python_interpreter//:python_bin",
-    python_version = "PY3",
+# <default py version change>: If the default version of python changes for Sematic, this
```

Comment:
> <default py version change>

Added this anywhere we'd need to change something if we change the default interpreter we use